fluentd/Dockerfile

```diff
@@ -32,6 +32,7 @@ COPY ${upstream_code}/lib ${HOME}/vendored_gem_src
 COPY ${upstream_code}/install-gems.sh ${HOME}/vendored_gem_src/
 COPY ${upstream_code}/*.patch.sh ${HOME}/vendored_gem_src/
 COPY ${upstream_code}/*.patch ${HOME}/vendored_gem_src/
+RUN rm ${HOME}/vendored_gem_src/log4366_ruby_lib_net_http.patch
```

Comment:
nit: will be useful for future to add some comment here because not clear why we copy patches and on next step remove this particular file